arclightning_backend/arclightning_controller.cfg

```diff
@@ -0,0 +1,30 @@
+﻿[General]
```

Comment:
y tho

Reply:
don't we want to allow for the same fight stick mappings to keyboard across devices?